src/steps/check-lighthouse-performance.ts

```diff
@@ -0,0 +1,56 @@
+import { BaseStep, Field, StepInterface } from '../core/base-step';
+import { Step, RunStepResponse, FieldDefinition, StepDefinition } from '../proto/cog_pb';
+
+export class CheckLighthousePerformance extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Check a page\'s Lighthouse performance score';
+  // tslint:disable-next-line:max-line-length
+  protected stepExpression: string = 'the (?<throttleTo>mobile|desktop) lighthouse performance score should be (?<expectedScore>\\d{1,3}) or higher';
+  protected stepType: StepDefinition.Type = StepDefinition.Type.VALIDATION;
+  protected expectedFields: Field[] = [{
+    field: 'throttleTo',
+    type: FieldDefinition.Type.STRING,
+    description: 'Throttle Level (mobile or desktop)',
+  }, {
+    field: 'expectedScore',
+    type: FieldDefinition.Type.NUMERIC,
+    description: 'Expected Score',
+  }];
+
+  async executeStep(step: Step): Promise<RunStepResponse> {
+    const stepData: any = step.getData().toJavaScript();
+    const throttleTo: 'mobile' | 'desktop' = stepData.throttleTo;
+    const expectedScore: number = stepData.expectedScore;
+
+    try {
+      const currentUrl = await this.client.getCurrentPageInfo('url');
+      const lhr = await this.client.getLighthouseScores(currentUrl.toString(), throttleTo);
+
+      const performance: any = Object.values(lhr.categories)[0];
+      const actualScore = performance.score * 100;
```

Comment:
- [ ] I have occasionally seen a score come back that yields a non-whole number (e.g. `28.999999999999996`).  We should round this.